src/crimson/net/chained_dispatchers.h

```diff
@@ -33,6 +30,9 @@ class ChainedDispatchers {
   void ms_handle_connect(crimson::net::ConnectionRef conn);
   void ms_handle_reset(crimson::net::ConnectionRef conn, bool is_replace);
   void ms_handle_remote_reset(crimson::net::ConnectionRef conn);
+
+ private:
+  std::list<Dispatcher*> dispatchers;
```

Comment:
wondering why not use `boost::container::small_vector<>` ?

Reply:
Thanks for recommendation. Fixed in https://github.com/ceph/ceph/pull/38360/commits/a5a77b518705123f51e15c4d9e9bb83c7969288d